tests/unit/controllers/new-signup-test.js

```
@@ -90,24 +90,36 @@ module('Unit | Controller | new-signup', function (hooks) {
-
-    controller.send('handleCheckboxInputChange', 'developer', false);
-    assert.false(
-      controller.signupDetails.roles.developer,
-      'Developer role unset',
-    );
-    assert.true(
-      controller.isButtonDisabled,
-      'Button disabled when no roles selected',
-    );
-  });
+test('handleCheckboxInputChange updates role and toggles button state', function (assert) {
```

ESLint

[error] Insert `··`

(prettier/prettier)